resources/js/components/dashboard/dashboard-main/dashboard.js

```diff
@@ -146,13 +146,26 @@ class Dashboard extends React.Component {
         usersData = [];
         upcomingDates = [];
 
-        axios.get('/hires').then(response => {
-            console.log(response);
+        axios.post('hires/search', {
+            "searchText": this.state.searchString,
+            "step": null,
```

Comment:
If we plan on using the step on searching, can we put something dynamic in there instead of null?